legend-engine-pure-code-compiled-core/src/main/resources/core/pure/binding/executionPlan/executionPlan_generation.pure

```diff
@@ -51,6 +51,7 @@ function meta::external::shared::format::executionPlan::processValueSpecificatio
       f:SimpleFunctionExpression[1]         | $f->getFunctionProcessor($extensions)->eval($f, $state, $extensions, $debug), 
       c:ClusteredValueSpecification[1]      | $c->plan($state.inScopeVars, $state.exeCtx, $extensions, $debug),
       e:ExtendedRoutedValueSpecification[1] | $e.value->processValueSpecification($state, $extensions, $debug),
+      var:VariableExpression[1]             | ^VariableResolutionExecutionNode(varName=$var.name, resultType=^ResultType(type=Any)),
```

Comment:
How hard will be to use the type of the variable expression to set the node type?  Same for the node multiplicity.

Reply:
This is actual no longer need. Have removed.